.github/workflows/rust.yml

```diff
@@ -40,6 +40,24 @@ jobs:
         env: ${{ matrix.env }}
         run: ./contrib/test.sh
 
+  Cross:
+    name: Cross testing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Crate
+        uses: actions/checkout@v2
+      - name: Checkout Toolchain
+        uses: actions-rs/toolchain@v1
```

Comment:
Can you convert this to `apt-get` lines so that we aren't relying on GitHub Actions proprietary stuff?

Reply:
The same actions are used in some lines above and below, I just added the target field

Reply:
Right, but the other actions uses are just "install rust via rustup", this I'm not 100% sure how it maps to `rustup` commands.

Reply:
@TheBlueMatt `profile`, `toolchain`, `override`, `target` are all rustup's flags.

Reply:
I think i can remove `target: s390x-unknown-linux-gnu`
and add a step with `rustup target add s390x-unknown-linux-gnu`

do you prefer like this?

Reply:
I'd appreciate it, yes.

Reply:
done